generators/server/templates/src/main/kotlin/package/domain/User.kt.ejs

```diff
@@ -87,9 +87,7 @@ import javax.persistence.JoinColumn
 import javax.persistence.JoinTable
 import javax.persistence.ManyToMany
 import javax.persistence.OneToMany
-    <%_ if (!(prodDatabaseType === 'mysql' || prodDatabaseType === 'mariadb')) { _%>
 import javax.persistence.SequenceGenerator
```

Comment:
Instead of this can we do `import java.persistence.*`  just like the main generator? @emilpaw  what do you think?

Reply:
We could do that but isn't the goal to get rid of the wildcard imports? Both detekt and ktlint don't allow wildcard imports in the default config.

Reply:
We do have a few, but didn't Kotlin relaxed the rule for wild card imports? I thought they did but I am not sure either. Lets have it like this itself then.

Reply:
AFAIK no. Removing the wildcard imports is also specified here: [!306](https://github.com/jhipster/jhipster-kotlin/issues/306) 

I generally think it's a good idea to get rid of them.